tests/test.javascript.js

```diff
@@ -0,0 +1,91 @@
+import { SEVERITY_ERROR, SEVERITY_WARNING } from 'const';
+import JavaScriptScanner from 'javascript';
+import { singleLineString } from 'utils';
+
+
+describe('JS Code Checker', function() {
+
+  it('should not allow mozIndexedDB', (done) => {
+    var code = 'var myDatabase = indexeddb || mozIndexedDB;';
+    var jsScanner = new JavaScriptScanner(code, 'badcode.js');
+
+    jsScanner.scan()
```

Comment:
If you return a promise in a test you don't need to use `done()` it's actually quite useful as this will expose issues where the code misses returning a promise too.